src/main/java/pika/Pika.java

```diff
@@ -0,0 +1,205 @@
+package pika;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+import javafx.application.Application;
+import javafx.fxml.FXML;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
```

Comment:
Maybe you could add an empty line here to separate between `import javafx` and `import pika` inorder to improve readability?